src/server/scripts/Outland/BlackTemple/boss_shade_of_akama.cpp

```diff
@@ -232,7 +233,7 @@ struct npc_akama_shade : public ScriptedAI
             {
             case POINT_ENGAGE:
                 me->SetHomePosition(me->GetPosition());
-                me->SetFaction(FACTION_ENGAGE);
+                me->SetFaction(FACTION_MONSTER_SPAR_BUDDY);
```

Comment:
For the record, Akama is supposed to change his faction (twice), not the Shade.

Reply:
This is akama, and not the shade. The naming is confusing